src/client/src/app-base/components/App.module.scss

```diff
@@ -1,3 +0,0 @@
-.mapContainer {
```

Comment:
we can remove the file if it's empty! Or unless you already did, I can't tell with the github UI...